src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionAT.java

```diff
@@ -388,6 +390,47 @@ public void whenStatsOnNotInitializedSubscriptionThanCorrectResponse() throws IO
         }
     }
 
+    @Test
+    public void whenStreamDuplicatePartitionsThenUnprocessableEntity() throws IOException {
+        final String et = createEventType().getName();
+        final Subscription s = createSubscriptionForEventType(et);
+
+        final String body = "{\"partitions\":[" +
+                "{\"event_type\":\"et1\",\"partition\":\"0\"}," +
+                "{\"event_type\":\"et1\",\"partition\":\"0\"}]}";
+        given().body(body)
+                .contentType(JSON)
+                .when()
+                .post("/subscriptions/{sid}/events", s.getId())
+                .then()
+                .statusCode(HttpStatus.SC_UNPROCESSABLE_ENTITY)
+                .body(JSON_HELPER.matchesObject(Problem.valueOf(
+                        MoreStatus.UNPROCESSABLE_ENTITY,
+                        "Duplicated partition specified")));
+    }
+
+    @Test
+    public void whenStreamWrongPartitionsThenUnprocessableEntity() throws IOException {
```

Comment:
Maybe it's overkill, but do you think it would be useful to have a test that tries to get partition `N + K`, from an event type with `N` partitions (where `K > 0`)?

Reply:
Yes, makes sense. I added that https://github.com/zalando/nakadi/pull/848/commits/b982da9499e7c05ee6672317178942fc5f72ee2c